wormhole-connect/src/views/v2/Bridge/AssetPicker/index.tsx

```diff
@@ -211,10 +214,12 @@ const AssetPicker = (props: Props) => {
             isFetching={props.isFetching}
             selectedChainConfig={chainConfig}
             selectedToken={props.token}
+            selectedTokenChain={selectedTokenChain}
```

Comment:
Maybe I'm missing a detail here, but isn't `selectedTokenChain.key` going to be the same as `selectedChainConfig.key` when a token is selected? Why do we have to set the selected token chain and pass it as a prop? Just trying to make sure I understand

Reply:
Naming should have been better, I agree. We can select a chain without selecting a token. `selectedChainConfig` is the currently selected chain in the asset picker. When user switch between chains without selecting a token `selectedChainConfig` is still set ot the currently selected chain on the screen.
In the other hand `selectedTokenChain` is the chain of the last selected token. It only updates when user clicks and selects a token.

Reply:
Maybe this explains better :)
https://www.loom.com/share/aafea555716349e68313a0390b0d1f87?sid=7c95d48a-eae2-4351-9df6-731f73b7ec83